### PR TITLE
[WIP] Feature multiple dims

### PR DESCRIPTION
**Pull request by mraspaud**
This PR fixes multidimension xarray.DataArray resampling

### CODE REVIEW

File: pyresample/kd_tree.py [hunk 1]
```diff
@@ -711,7 +726,7 @@ def get_sample_from_neighbour_info(resample_type, output_shape, data,
         if is_multi_channel:
             weight_funcs = weight_funcs * 2
         else:
-            weight_funcs = (weight_funcs,) * 2
+            weight_funcs = (weight_funcs, ) * 2
```
Comment (davidh-ssec) [inline pyresample/kd_tree.py]: What linter prefers this? Destroy it.

Comment (mraspaud) [reply pyresample/kd_tree.py]: it also says `r**2 / b**2` is better than `r ** 2 / b ** 2`, which I kind of like

Comment (mraspaud) [reply pyresample/kd_tree.py]: It's actually no a linter, it's yapf

**Comment by davidh-ssec** [comment]
@mraspaud Is this ready for merge?

**Comment by mraspaud** [comment]
Not yet, the performance is pathetic.

**Pull request merged**